pad_crm/models.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2016 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+# Copyright 2019 Artem Rafailov <https://it-projects.info/team/Ommo73/>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html)
+from odoo import fields, models
```

Comment:
You renamed the file, why? It wasn't necessary. If you decided to update the repository file hierarchy according to the present rules then you need to place the file into the `models` directory